projects/plugins/jetpack/jsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+	"compilerOptions": {
+		"jsx": "react",
+		"paths": {
+			// fallback to resolving imports relative to _inc/client
+			"*": [ "./*", "./_inc/client/*" ]
```

Comment:
If you don't add `"@automattic/jetpack-*": [ "../../js-packages/*" ]` here, VS Code can't find package definitions

e.g. in `projects/plugins/jetpack/_inc/client/components/connect-button/index.jsx`

<img width="1307" alt="Screenshot 2022-04-06 at 10 13 18 PM" src="https://user-images.githubusercontent.com/18226415/162025583-f73dc628-86c3-4c5c-9f6e-30c638dba3e0.png">

Reply:
Looks like that's just whining about the JS package not having TypeScript ".d.ts" type definitions. :shrug: The actual "Go to Definition" feature works, and your suggestion breaks the package boundaries.

Looking through some search results, it seems like the vscode developers consider this "working as intended", see for example https://github.com/microsoft/TypeScript/issues/34271.

Reply:
We may need to do this when we switch to TypeScript.

Reply:
If we do switch to typescript at some point, the solution then would be to make the ".d.ts" files rather than the suggestion here.